### ISSUE TITLE
GPUDevice: uncapturederror event seems not implemented

### ISSUE THREAD

**Issue by disjukr** [denoland/deno#22827]
Version: Deno 1.41.1

https://developer.mozilla.org/en-US/docs/Web/API/GPUDevice/uncapturederror_event

`test.ts`
```ts
const adapter = await navigator.gpu.requestAdapter();
const device = await adapter.requestDevice();
device.addEventListener("uncapturederror", (e) => console.log(e));
```

```sh
$ deno run --unstable-webgpu test.ts
error: Uncaught (in promise) TypeError: Cannot destructure property 'listeners' of 'self[eventTargetData]' as it is undefined.
device.addEventListener("uncapturederror", (e) => console.log(e));
       ^
    at GPUDevice.addEventListener (ext:deno_web/02_event.js:931:13)
    at test.ts:3:8
```

**Comment by bartlomieju** [comment]
CC @crowlKats

**Comment by crowlKats** [comment]
Closing as implemented in #23684
